assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -105,6 +112,28 @@ const DashboardIdeasWidget = ( {
 		[ setHash, setActiveTabIndex ]
 	);
 
+	// Any time the pagination value changes, scroll to the top of the container.
+	useUpdateEffect( () => {
+		const tabBarRectangle = tabBarHeaderRef.current.getBoundingClientRect();
+
+		const isOnScreen =
+			tabBarRectangle.top >= 0 &&
+			tabBarRectangle.left >= 0 &&
+			tabBarRectangle.bottom <=
+				( global?.innerHeight ||
+					global?.document?.documentElement?.clientHeight ) &&
+			tabBarRectangle.right <=
+				( global?.innerWidth ||
+					global?.document?.documentElement?.clientWidth );
```

Comment:
Are all these conditional chains needed?

Reply:
_Probably not, but I'm always wary of accessing things off the `global`/`window`. But these should be fine 😄